src/index.js

```diff
@@ -1,6 +1,9 @@
 'use strict';
 
 require('./assets/markdown.css');
```

Comment:
The files `src/assets/styles.css`, `src/assets/header.css` and `src/assets/navigation.css` are generate by elm-css, why do we need to add them on git?

In the [lib's example](https://github.com/rtfeldman/elm-css/tree/master/examples) they don't add the generated files to git.